src/repo/learnocaml_exercise.ml

```diff
@@ -179,6 +183,29 @@ module File = struct
       update = (fun solution ex -> { ex with solution })
      }
 
+  let depend =
+    { key = "../depend.txt" ; ciphered = false ; (* does'nt work with ciphered = true *)
```

Comment:
The comment is not correct english and is really cryptic. Please explain to the reader what is at stake here or make it explicit in the code.